bot/commands/misc/avatar.js

```diff
@@ -0,0 +1,20 @@
+// Shows the avatar of the user provided
+
+const {Command} = require('yuuko');
+const {parseUser} = require('../../util/discord');
+
+module.exports = new Command('avatar', async (msg, args) => {
+	if (args.length === 0) msg.channel.createMessage(msg.author.avatarURL).catch(() => {});
+	const [member] = await parseUser(args.join(' '), msg.channel.guild, msg.author);
+	if (!member) {
+		msg.channel.createMessage('Member not found, check if what you\'re using is correct or try using an ID.').catch(() => {});
```

Comment:
Use double quotes to avoid escaping the apostrophe in this string (i know, it's a weird convention, but i'm pretty sure eslint enforces it)
```suggestion
		msg.channel.createMessage("Member not found, check if what you're using is correct or try using an ID.").catch(() => {});
```

Reply:
this isn't actually a thing, i'm misremembering my own eslint configuration lol